trail/query/templates/query.html

```diff
@@ -1,19 +1,54 @@
 {% extends "base.html" %}
+{% load static %}
+
 
 {% block query-nav-state%}active{% endblock %}
 
+
+{% block head.extend %}
+<style>
+ table, th, td {
+     border: 1px solid black;
+ }
+</style>
+{% endblock %}
+
+
 {% block content %}
 <!-- Post -->
 <section class="post">
 		<header class="major">
 				<h1>This is where <br />
 						the query lives</h1>
-				<p>Aenean ornare velit lacus varius enim ullamcorper proin aliquam<br />
-						facilisis ante sed etiam magna interdum congue. Lorem ipsum dolor<br />
-						amet nullam sed etiam veroeros.</p>
+				<p>The placeholder is removed here.<br />
+						Here as well.<br />
+						So as here.</p>
```

Comment:
Please put some useful text here. The `h1` header could read "Query Tool" and a single paragraph could read, for example, "Query the metadata database generated from previously uploaded FITS files below."

Reply:
Fixed